releng/platform-definitions/platform-definition-2022-09/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-   Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
-   Copyright (c) 2022, Datadog, Inc. All rights reserved.
+   Copyright (c) 2022 Oracle and/or its affiliates. All rights reserved.
+   Copyright (c) 2022 Datadog, Inc. All rights reserved.
```

Comment:
I think here the resolve wasn't correct. Copyright should now be `2022, 2023,` after my last change to the file. Please fix.